airmozilla/starred/static/starred/js/star_event.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function(){
```

Comment:
Use the `$(function() {` notation instead. You'll find it being used in many of the other jquery-using files in the project.